app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+
+    if len(parts) != 3 or parts[0] != "cp" or parts[1] == parts[2]:
+        raise ValueError("Invalid command format")
+
+    try:
+        with open(parts[1], "r") as file, open(parts[2], "w") as copy:
```

Comment:
Code Style: It's recommended to use more descriptive variable names. For example, instead of `file` and `copy`, you could use `source_file` and `destination_file`. This will make your code easier to read and understand.